impeller/entity/contents/filters/gaussian_blur_filter_contents.cc

```diff
@@ -0,0 +1,273 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "impeller/entity/contents/filters/gaussian_blur_filter_contents.h"
+
+#include "impeller/entity/contents/content_context.h"
+#include "impeller/entity/texture_fill.frag.h"
+#include "impeller/entity/texture_fill.vert.h"
+#include "impeller/renderer/command.h"
+#include "impeller/renderer/render_pass.h"
+#include "impeller/renderer/sampler_library.h"
+
+namespace impeller {
+
+using GaussianBlurVertexShader = GaussianBlurPipeline::VertexShader;
+using GaussianBlurFragmentShader = GaussianBlurPipeline::FragmentShader;
+
+namespace {
+SamplerDescriptor MakeSamplerDescriptor(MinMagFilter filter,
+                                        SamplerAddressMode address_mode) {
+  SamplerDescriptor sampler_desc;
+  sampler_desc.min_filter = filter;
+  sampler_desc.mag_filter = filter;
+  sampler_desc.width_address_mode = address_mode;
+  sampler_desc.width_address_mode = address_mode;
```

Comment:
you set the width address mode twice here, does this need to be height address mode too? I wonder if this is causing the artifacts.

Reply:
I checked and this did not fix the artifacting issue. I suspect this is probably to do with coverage then

Reply:
Good catch, that definitely would have shown up at some point.  The default value is nearest.  Maybe that was contributing to the worse waffling we were seeing with the 2 step?